test/expected/build/library.j/dest/resources/library/j/dependency-es6-1.js

```diff
@@ -9,9 +9,24 @@
  * - ClassDeclaration
  */
 (sap?.ui).define([`Bar`], (Bar) => {
-	return class Foo extends Bar {
+	/**
+	 * @class
+	 * My super documentation of this class
+	 *
+	 * @extends library.j.Bar
+	 *
+	 * @author SAP SE
+	 * @version ${version}
+	 *
+	 * @public
+	 * @alias library.j.Foo
+	 * @ui5-metamodel text
```

Comment:
Please remove. This tag is about to be removed.

Reply:
Thanks! Removed it